rpcs3/rpcs3qt/debugger_list.cpp

```diff
@@ -329,27 +335,30 @@ void debugger_list::wheelEvent(QWheelEvent* event)
 
 void debugger_list::resizeEvent(QResizeEvent* event)
 {
-	Q_UNUSED(event)
+	QListWidget::resizeEvent(event);
 
 	if (count() < 1 || visualItemRect(item(0)).height() < 1)
 	{
 		return;
 	}
 
-	m_item_count = (rect().height() - frameWidth() * 2) / visualItemRect(item(0)).height();
+	const u32 old_size = m_item_count;
 
-	clear();
+	// It is fine if the list is a tad bit larger than the list
+	m_item_count = utils::aligned_div<u32>(rect().height() - frameWidth() * 2, visualItemRect(item(0)).height());
 
-	for (u32 i = 0; i < m_item_count; ++i)
+	if (old_size <= m_item_count)
 	{
-		insertItem(i, new QListWidgetItem(""));
+		for (u32 i = old_size; i < m_item_count; ++i)
+		{
+			insertItem(i, new QListWidgetItem(""));
+		}
 	}
-
-	if (horizontalScrollBar())
+	else
 	{
-		m_item_count--;
-		delete item(m_item_count);
+		for (u32 i = old_size - 1; i >= m_item_count; --i)
+		{
+			delete item(i);
```

Comment:
I think this is a bug
```suggestion
			delete takeItem(i);
```